README.md

```diff
@@ -0,0 +1,67 @@
+This repository contains standardized interface descriptions for aircraft cabin equipment.
+Here, the aerospace industry collaborates to shape the cabin network of the future.
+
+
+## Introduction
+
+This repository complements the ARINC 853 "CSMIM" standard. ARINC 853 specifies a modern
+communication protocol for non-essential cabin equipment. The protocol specification draws
+from related standards in the building and home automation domain, including
+[MQTT](https://mqtt.org/) and [CBOR](https://cbor.io/) as building blocks. Whereas the
+ARINC 853 document describes the generic protocol and data structure, this
+repository &ndash; the CSMIM Knowledge Base &ndash; contains concrete interface definitions for
+various cabin equipment. These definitions are considered "standard" and should be used
+by all cabin equipment suppliers as far as possible.
+
+The goal of our activities is to enable interoperability of all cabin equipment on all
+types of aircraft, and to create an open aircraft cabin network where any supplier can
+add new, useful functions.
+
+
+## How to get started
+
+You should have a basic understanding of the ARINC 853 standard itself, in particular of its
+chapter 6.2 which describes the CSMIM abstract data model. This repository primarily contains
+a list of standardized *CSMIM object types*, which are specified in ARINC 853 §6.2.7. The
+object types are located in the [types/ folder](types/) as files in
+[YAML](https://en.wikipedia.org/wiki/YAML) format.
+
+- **If you search for object types:**
+  Use the "Go to file" search box and input the object type
+  identifier or simply phrases of interest. For example, input "light" to find object types
+  concerned with cabin lighting. The search results will include the types
+  (`types/csmim.obj.*.yaml`) plus all potential locations for objects instantiated from
+  these types (`path/.../csmim.obj.*.yaml`).
+
+- **If you could not find an object type matching your requirements:**
+  Try searching the [issues](https://github.com/ARINC-IA/CSMIM/issues) for your requirements.
+  Maybe someone is already working on your topic, and you can join an existing discussion.
+  If the topic is new: Create an issue, describe your requirements and label it a "question".
+
+- **If you want to propose a new object type or enhance an existing one:**
+  Create an issue, describe your requirements and label it an "enhancement". If there are
+  existing object types similar to what you need, mention them and explain what is
+  missing for you and how the type should be enhanced.
+
+  If you have a very concrete proposal, you can fork this repository and add
+  the YAML file specifying your object type. Never edit an existing file &ndash;
+  once standardized, an object type cannot be changed. Always create a new file,
+  possibly with the same name and increased version number. Then create a pull
+  request.
+
```

Comment:
do we want to add a remark that manufacturer specific additions could me managed in forked repositories? or should we concentrate on the standard here?

Reply:
I added a paragraph to this avail.